webapp/src/components/App/Singlestudy/explore/TableModeList/dialogs/TableTemplateFormDialog.tsx

```diff
@@ -47,16 +58,8 @@ function TableTemplateFormDialog(props: TableTemplateFormDialogProps) {
             autoFocus
             control={control}
             rules={{
-              validate: (value) => {
-                const id = getValues("id");
-                const hasDuplicate = templates.find(
-                  (tp) => tp.id !== id && tp.name.trim() === value.trim(),
```

Comment:
The field name is break in update modal by removing `id` check, same name will not be allowed

Reply:
thanks I will check this